lib/dotcom_web/components/route_symbols.ex

```diff
@@ -149,13 +165,46 @@ defmodule DotcomWeb.Components.RouteSymbols do
       type="icon-svg"
       name={"icon-logan-express-#{@route.name}"}
       class={"#{@class} #{@cva_class}"}
+      aria-label={route_label(@route)}
+      role="text"
     />
     """
   end
 
   def route_icon(assigns) do
     ~H"""
-    <.icon type="icon-svg" name="icon-mode-shuttle-default" class={"#{@class} #{@cva_class}"} />
+    <.icon
+      type="icon-svg"
+      name="icon-mode-shuttle-default"
+      class={"#{@class} #{@cva_class}"}
+      aria-label="Shuttle"
+      role="text"
+    />
     """
   end
+
+  # Given a route, return a machine-readable label.
+  defp route_label(%Route{description: :ferry}), do: "Ferry"
+  defp route_label(%Route{description: :regional_rail}), do: "Commuter Rail"
+  defp route_label(%Route{external_agency_name: "Logan Express"}), do: "Logan Express"
```

Comment:
By the way I appreciate the attentiveness to trying to match what the icon communicates visually :)